functests/1_performance/cpu_management.go

```diff
@@ -290,12 +291,17 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			containerID, err := pods.GetContainerIDByName(testpod, "test")
 			Expect(err).ToNot(HaveOccurred())
 
-			cmd := []string{"/bin/bash", "-c", fmt.Sprintf("find /rootfs/sys/fs/cgroup/cpuset/ -name *%s*", containerID)}
-			containerCgroup, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
-			Expect(err).ToNot(HaveOccurred())
+			containerCgroup := ""
+			Eventually(func() bool {
+				cmd := []string{"/bin/bash", "-c", fmt.Sprintf("find /rootfs/sys/fs/cgroup/cpuset/ -name *%s*", containerID)}
+				containerCgroup, err = nodes.ExecCommandOnNode(cmd, workerRTNode)
+				Expect(err).ToNot(HaveOccurred())
+				return containerCgroup != ""
+			}, (cluster.ComputeTestTimeout(30*time.Second, RunningOnSingleNode)), 5*time.Second).ShouldNot(BeEmpty(),
```

Comment:
Please update to `Should(BeTrue())` or just return `return containerCgroup`

Reply:
sure, done